tests/unit/test_io_hdf5_h5tools.py

```diff
@@ -1017,39 +1071,35 @@ def test_io_read_herd_value_warn(self):
 
         self.remove_er_files()
 
-    def test_io_write_herd(self):
-        er = HERD()
-        self.foofile.link_resources(er)
-
-        data = Data(name="species", data=['Homo sapiens', 'Mus musculus'])
-        er.add_ref(file=self.foofile,
-                   container=data,
-                   key='key1',
-                   entity_id='entity_id1',
-                   entity_uri='entity1')
+    @unittest.skipIf(not LINKML_INSTALLED, "optional LinkML module is not installed")
+    def test_io_write_extend_herd(self):
```

Comment:
@rly This is the extend test

Reply:
Got it. That's what I was pointing to, but for some reason github showed a weird location in the diff.